tutorial-content/content/weighting-schemes.md

```diff
@@ -290,6 +345,16 @@ write aggregate by a factor of $10$ for it to match the PRISM data:
 landscan <- aggregate(landscan, fact=10, fun=sum)
 ```
 ````
+````{tab-item} Python
+```Python
+# Dimensions for the aggregated data
+new_shape = (landscan.shape[0] // 10, landscan.shape[1] // 10)
+
+# Use numpy to reshape and aggregate by summing
+reshaped = landscan.reshape((new_shape[0], 10, new_shape[1], 10))
+aggregated_landscan = reshaped.sum(axis=1).sum(axis=2)
```

Comment:
Shouldn't it be summing over axes 1 and 3? Can you test this?

Reply:
Oh, I think I understand how it ends up being axis=1 and axis=2. After you sum over axis 1, axis 3 moves up a spot.